pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -929,6 +929,39 @@ func GetOpenshiftBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: TemplateServiceBrokerRoleName,
+				Annotations: map[string]string{
+					roleSystemOnly: roleIsSystemOnly,
+				},
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					APIGroups: []string{authorizationapi.LegacyGroupName},
+					Verbs:     sets.NewString("create"),
+					Resources: sets.NewString("subjectaccessreviews"),
+				},
+				{
+					APIGroups: []string{templateapi.GroupName},
+					// "impersonate" is required for the API server to accept creation of
```

Comment:
How about verb=assign, group=templateapi.GroupName resource=templateinstances .  You could then bind that role locally in a namespace to allow subdivision or clusterwide if you (cluster-admin) don't care about subdivision.